shared/teams/team/invites/invite-history.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Container from '../../../util/container'
+import * as Constants from '../../../constants/teams'
+import * as Types from '../../../constants/types/teams'
+import * as TeamsGen from '../../../actions/teams-gen'
+import {memoize} from '../../../util/memoize'
+import {Section} from '../../../common-adapters/section-list'
+import {formatDurationLong} from '../../../util/timestamp'
+import {useTeamDetailsSubscribe} from '../../subscriber'
+import {ModalTitle} from '../../common'
+
+type Props = Container.RouteProps<{teamID: Types.TeamID}>
+
+const splitInviteLinks = memoize((inviteLinks: Set<Types.InviteLink>): {
```

Comment:
when I click "expire link" it doesn't move to the other section, it just vanishes.

Reply:
known, ticketed separately